### ISSUE TITLE
Why My Build on Linux is CPU-Only?

### ISSUE THREAD

**Issue by yefeng-zheng** [Microsoft/CNTK#182]
I am trying to setup CNTK on a Ubuntu machine. I followed the instruction on wiki step-by-step and successfully installed all dependent libraries. To verify, I downloaded and tested the pre-built CNTK package. Everything worked well.

I then started to build CNTK from source code. After running ../../configure, I got the following print-out. Nothing is suspicious.

_Defaulting to --with-buildtype=release
Found cuda at /usr/local/cuda-7.5
Found gdk at /usr/.
Found CUB at /usr/local/cub-1.4.1
Found cuDNN at /usr/local/cudnn-4.0
Found OpenCV at /usr/local/opencv-3.0.0
Found libzip at /usr/local
Generating /home/hchen/CNTK/CNTK-master/build/release/Config.make
Generating /home/hchen/CNTK/CNTK-master/build/release/Makefile
run
>make -j all
to build_

I then built the project. The compiling was very fast (taking only a few seconds). When I ran CNTK, the print-out told me "Build target: CPU-only" and it only used CPU for training.

Anybody can help me to build a GPU-enabled version?

**Comment by eldakms** [comment]
We cannot reproduce it in our environment. The suspicion is that the cuda path is not set up correctly (we usually use 7.0)

Could you please attach your config from /home/hchen/CNTK/CNTK-master/build/release/Config.make and probably also the buildinfo.h from /home/hchen/CNTK/CNTK-master/Sources/CNTK/buildinfo.h

**Comment by yefeng-zheng** [comment]
Thanks for the hint. I checked file buildinfo.h. It says "Build target: CPU-only." I recalled that in the first try, I forgot to install nVidia gtk and resulted in a CPU-only build. I then installed gtk and ran the config again. Due to some reason, file buildinfo.h wasn't generated again. That's why I always got a CPU-only build.

I started everything  from scratch by checking out CNTK to a different folder. The first build failed because file buildinfo.h hadn't been generated yet. The second build succeeded. Now, I have a GPU build too.

BTW - I have both CUDA 7.0 and 7.5 installed on my computer. It has no problem to build with 7.5.
